src/server/json_family.cc

```diff
@@ -157,6 +159,48 @@ OpResult<json> GetJson(const OpArgs& op_args, string_view key) {
   return decoder.get_result();
 }
 
+string ConvertToJsonPointer(const string& json_path) {
+  if (json_path.empty() || json_path[0] != '$') {
+    return "";
+  }
+
+  vector<string> parts;
+  bool invalid_format = false;
+  std::size_t i = 1;
```

Comment:
please add a comment saying you assume the script has a valid format. (you assume that '[' is followed up by a character). Also, maybe I am missing something but do you expect`invalid_format` to be true?

If it should not happen, I prefer using LOG`(FATAL)` instead of silently returning an empty string.